scripts/mvcjsoninput.2mb.lua

```diff
@@ -27,12 +27,9 @@ function init(args)
 
     data = data .. "]"
 
-   local contentLength = string.len(data)
-
    wrk.method = "POST"
    wrk.body = data
    wrk.headers["Content-Type"] = "application/json"
-   wrk.headers["Content-Length"] = contentLength
```

Comment:
this was incorrect and causing the server to throw due to a mismatch. wrk seemingly figures out the right value by itself.